xdsl/dialects/csl/csl.py

```diff
@@ -1311,7 +1313,7 @@ class SetDsdStrideOp(IRDLOperation):
 
     name = "csl.set_dsd_stride"
     op = operand_def(DsdType)
-    stride = operand_def(IntegerType(8, Signedness.SIGNED))
+    stride = operand_def(eq(i8) | eq(IntegerType(8, Signedness.SIGNED)))
```

Comment:
Might as well also declare this type locally for consistency
```suggestion
    stride = operand_def(eq(i8) | eq(si8))
```